1904-10-31.xml

```diff
@@ -0,0 +1,2512 @@
+                        sovereigns throw their personal influence into the scale, and diplomacy is
+                        called into play, with the combined result of pouring oil on troubled waters
+                        and easing a tension which in earlier times must inevitably have led to
+                        hostilities. We should be sorry to think that the old spirit, which would
+                        not brook an insult to England on the part of a foreigner, has become
+                        softened by a higher state of culture, and that we would to-day clamly
+                        submit to see the honour of our nation outraged one whit more than our
+                        forefathers would have done. This feeling still lives, and our telegrams
+                        have shown that the entire nation is at one in demanding reparation for what
+                        was not only a blunder but a dsatradly crime. The general consensus of
+                        opinion, as evinced by the great London daily newspapers, show that the
+                        solution of this difficulty has been received with satisfaction. It is to be
+                        hoped, therefore, that the wrong will be fully recognised by the arbitrators
+                        and that the cuase of international comity will be served by the
+                        righteousness of their award.</p>
+                </div>
+                <div type="item">
+                    <head>AGRICULTURAL DEPARTMENT WANTED.</head>
+                    <p>A short time ago a rumor was started to the effect that the Egyptian
+                        Government was about to form and Agricultural Department, but the report was
+                        found to have been merely based on the fact that an increased grant was to
+                        be made to the Khedivial Agricultural Society. It is much to be regretted
+                        that the Government should have failed to have boldly faced the pressing
+                        necessities of the situaution, in view of the very large amount of funds now
+                        placed at its disposal. We could have wished that agriculture ahould have
+                        received a larger share of administrative attention than heretofore, as for
+                        years past we have persistently advocated in these columns that expediency
+                        of giving effect to its claims. Afterall, husbandry in Egypt is an industry
+                        which in magnitude and importance immeasurably transcends all other. The
+                        expenditure, therefore, connected with the establishment of a department
+                        exclusively devoted to the fostering and development of its interests would
+                        be money profitably employed. "Agriculture" in its purely technical sense is
+                        restricted</p>
```

Comment:
You don't need the paragraph or div breaks. you can just say `restricted <cb n="6"/> to the`